src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -241,6 +253,8 @@ def __init_basic_command_line_for_exec_file(self, test_command_line: list):
                 is_input_folder = True
                 command += f" --input_folders="
                 argument = argument[argument.find("=")+1:]
+            elif "--gtest_filter" in argument:
```

Comment:
`--gtest_filter` is set in class constructor. Looks like extra construction

Reply:
not applicable